### ISSUE TITLE
copy paste rich text in classic editor

### ISSUE THREAD

**Issue by oferlaor** [mcguffin/the-paste#66]
I'm realizing that the copy/paste features that were added to version 2.X are causing problems when pasting html from the current document (or another document from the current server). We do this constantly, and there are weird issues that occur when you do that.

could you add a button to turn on/off this feature (or maybe even enable/disable it globally)?

I've just reverted to 1.1.2 and most of these issues just go away, the built-in copy paste works perfectly.

I'm trying the "prefer document" button on/off but the issues (such as moving an image with a figcaption) remain in both options.

**Comment by mcguffin** [comment]
2.1.3 will come with a toggle option in the toolbar, allowing you to completely disable pasting.
<img width="204" alt="grafik" src="https://github.com/user-attachments/assets/cfeed4d8-f2f3-45a3-b427-1722e6f639a8" />

**Comment by oferlaor** [comment]
@mcguffin first, thanks!

I installed it, for some reason the new toolbar icon is "empty". this is how it looks on advanced editor tools:
<img width="597" alt="image" src="https://github.com/user-attachments/assets/9de737cd-cf42-41f8-9a2e-8a739d78205c" />

BTW, I'm not sure this is what I was after. I never want to turn off ThePaste completely. What I want to to do is to turn off all document related features. Meaning that when you copy/paste HTML or rich text of any kind, it would behave like version 1.X (it wouldn't do anything). Some paste operations include both rich text and images and the "prefer files" doesn't always work as expected, particularly when copying directly from Explorer/Finder.

**Comment by amccaugh** [comment]
I agree, the document-related features cause issues -- for instance, try pasting nested bullet points into another set of nested bullet points with The Paste enabled.  As @oferlaor mentioned, it would be nice to be able to disable all the more advanced document-pasting features and paste as in 1.x
